doc/user/builders-writing.xml

```diff
@@ -188,9 +116,8 @@ bld = Builder(action = 'foobuild &lt; $SOURCE &gt; $TARGET')
     <scons_example name="builderswriting_ex1">
        <file name="SConstruct">
 bld = Builder(action = 'foobuild &lt; $SOURCE &gt; $TARGET')
-env = Environment(BUILDERS = {'Foo' : bld})
-import os
-env['ENV']['PATH'] = env['ENV']['PATH'] + os.pathsep + os.getcwd()
+env = Environment(BUILDERS={'Foo': bld})
+env.AppendENVPath('PATH', os.getcwd())
```

Comment:
Don't we need import os here?

Reply:
not sure how it got dropped - or why the example builds without it.

Reply:
I checked all the examples, found a couple of other places with the same issue, and some now-unused example files to removed.